### PR TITLE
Allow empty mail config

### PR DESCRIPTION
**Pull request by alexisLefebvre**
Fix after #22.

### CODE REVIEW

File: behat.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    suites:
+        default:
+            paths:
+                features: src/Troopers/BehatContexts/Tests/Features/
+            filters:
+                tags: "@default"
+            contexts:
+                - Troopers\BehatContexts\Tests\Features\Context\TestContext
+    extensions:
+        Knp\FriendlyContexts\Extension:
```
Comment (Charlie-Lucas) [inline behat.yml:11]: Is it necessary?

File: behat.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+default:
```
Comment (Charlie-Lucas) [inline behat.yml:1]: pls move in Tests directory

File: behat.yml (hunk 1 above)
Comment (alexisLefebvre) [reply behat.yml:11]: Yes, without this I have an exception:

> [Behat\Testwork\ServiceContainer\Exception\ExtensionInitializationException]  
> Missing extension Knp\FriendlyContexts\Extension

File: behat.yml (hunk 2 above)
Comment (alexisLefebvre) [reply behat.yml:1]: I'll move the other file (`behat_config.yml`).

**Pull request merged**